networking/src/ApiClient.ts

```diff
@@ -757,9 +756,20 @@ export class GameSessionParser {
         return []
     }
 
-    private static parseAnswerType<t>(input: any | t): t {
+    private static convertProperty<T>(obj: any, propertyName: string, enumType: T) {
```

Comment:
parses the `AnswerSettings` AWSJSON so that `answerType` and `answerPrecision` values are extracted and mapped to `AnswerType` and `AnswerPrecision` enums, respectively.